internal/cmd/common/common.go

```diff
@@ -40,16 +40,11 @@ func CheckIfSigningViableWithFlag(cmd *cobra.Command, _ []string) error {
 		return fmt.Errorf("required flag \"signing-key\" not set")
 	}
 
-	return CheckIfSigningViable(cmd, nil)
+	return CheckIfSigningViable(nil, nil)
```

Comment:
I'd keep this in in case this ever changes. It's going to be dropped by the receiver anyway.